File: implementation/scipamato/core/entity/src/test/java/ch/difty/scipamato/core/auth/RolesTest.java
package ch.difty.scipamato.core.auth;

import static ch.difty.scipamato.core.auth.Roles.*;
import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class RolesTest {

    @Test
    void assertRoleName_User() {
        assertThat(USER).isEqualTo("ROLE_USER");
    }

    @Test
    void assertRoleName_Admin() {
        assertThat(ADMIN).isEqualTo("ROLE_ADMIN");
    }

    @Test
    void assertRoleName_Viewer() {
        assertThat(VIEWER).isEqualTo("ROLE_VIEWER");
    }

    @Test
    void dummyTest() {
        assertThat(dummyMethod()).isNull();
    }

}

File: implementation/scipamato/core/entity/src/test/kotlin/ch/difty/scipamato/core/auth/RolesTest.kt
package ch.difty.scipamato.core.auth

import ch.difty.scipamato.core.auth.Roles.*
import org.assertj.core.api.Assertions.assertThat

import org.junit.jupiter.api.Test

internal class RolesTest {

    @Test
    fun assertRoleName_User() {
        assertThat(USER).isEqualTo("ROLE_USER")
    }

    @Test
    fun assertRoleName_Admin() {
        assertThat(ADMIN).isEqualTo("ROLE_ADMIN")
    }

    @Test
    fun assertRoleName_Viewer() {
        assertThat(VIEWER).isEqualTo("ROLE_VIEWER")
    }

    @Test
    fun dummyTest() {
        assertThat(dummyMethod()).isNull()
    }

}
